ask-sdk-s3-persistence-adapter/README.md

```diff
@@ -0,0 +1,57 @@
+ASK SDK S3 Persistence Adapter package contains implementation of persistence adapter in Core SDK ('ask-sdk-core') based on AWS SDK.
+
+## Note
+
+Because Amazon S3 provides [eventual consistency](https://docs.aws.amazon.com/AmazonS3/latest/dev/Introduction.html) for updates to existing objects, we recommend using [ask-sdk-dynamodb-persistence-adapter](https://github.com/tianrenz/alexa-skills-kit-sdk-for-nodejs/tree/2.0.x/ask-sdk-dynamodb-persistence-adapter) for persistent attributes if your skill requires read-after-write consistency.
+
+## What is ASK SDK v2 for Node.js
+
+The ASK SDK v2 for Node.js is an open-source Alexa Skill Development Kit. ASK SDK v2 for Node.js makes it easier for you to build highly engaging skills, by allowing you to spend more time on implementing features and less on writing boiler-plate code.
+
+## Installing
+ASK SDK S3 Persistence Adapter package is an addon package for the core SDK ('ask-sdk-core') and thus has peer dependency of the core SDK package. From within your NPM project, run the following commands in the terminal to install them:
+
+```
+npm install --save ask-sdk-core
+```
+
+```
+npm install --save ask-sdk-s3-persistence-adapter
+```
+
+## Usage and Getting Started
+
+You can find a getting started guide [here](https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/wiki).
+
+## Usage with TypeScript
+The ASK SDK S3 Persistence Adapter package for Node.js bundles TypeScript definition files for use in TypeScript projects and to support tools that can read .d.ts files. Our goal is to keep these TypeScript definition files updated with each release for any public api.
+
+### Pre-requisites
+Before you can begin using these TypeScript definitions with your project, you need to make sure your project meets a few of these requirements:
+- Use TypeScript v2.x
+- Includes the TypeScript definitions for node. You can use npm to install this by typing the following into a terminal window:
+
+```
+npm install --save-dev @types/node
+```
+
+### In Node.js
+To use the TypeScript definition files within a Node.js project, simply import ask-sdk-dynamodb-persistence-adapter as below:
```

Comment:
typo : This should be ``ask-sdk-s3-persistence-adapter``. Also the code blocks below